QRInvite is not working

Here is a error of QRInvite cog when running `[p]qrinvite` command :

```Exception in command 'qrinvite'
Traceback (most recent call last):
  File "/home/pi/Red/lib/python3.7/site-packages/discord/ext/commands/core.py", line 79, in wrapped
    ret = await coro(*args, **kwargs)
  File "/home/pi/Blezo/cogs/CogManager/cogs/qrinvite/qrinvite.py", line 63, in qrinvite
    eextention = pathlib.Path(image_url).parts[-1].replace(".", "?").split("?")[1]
  File "/home/pi/.pyenv/versions/3.7.2/lib/python3.7/pathlib.py", line 994, in __new__
    self = cls._from_parts(args, init=False)
  File "/home/pi/.pyenv/versions/3.7.2/lib/python3.7/pathlib.py", line 649, in _from_parts
    drv, root, parts = self._parse_args(args)
  File "/home/pi/.pyenv/versions/3.7.2/lib/python3.7/pathlib.py", line 633, in _parse_args
    a = os.fspath(a)
TypeError: expected str, bytes or os.PathLike object, not Asset

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/pi/Red/lib/python3.7/site-packages/discord/ext/commands/bot.py", line 863, in invoke
    await ctx.command.invoke(ctx)
  File "/home/pi/Red/lib/python3.7/site-packages/discord/ext/commands/core.py", line 728, in invoke
    await injected(*ctx.args, **ctx.kwargs)
  File "/home/pi/Red/lib/python3.7/site-packages/discord/ext/commands/core.py", line 88, in wrapped
    raise CommandInvokeError(exc) from exc
discord.ext.commands.errors.CommandInvokeError: Command raised an exception: TypeError: expected str, bytes or os.PathLike object, not Asset```

Got an Issue too:

[2019-08-22 22:45:18] [ERROR] red: Exception in command 'qrinvite'
Traceback (most recent call last):
  File "/home/pi/.pyenv/versions/Red/lib/python3.7/site-packages/discord/ext/commands/core.py", line 79, in wrapped
    ret = await coro(*args, **kwargs)
  File "/mnt/extHDD/Red-DiscordBot/cogs/CogManager/cogs/qrinvite/qrinvite.py", line 63, in qrinvite
    eextention = pathlib.Path(image_url).parts[-1].replace(".", "?").split("?")[1]
  File "/home/pi/.pyenv/versions/3.7.2/lib/python3.7/pathlib.py", line 994, in __new__
    self = cls._from_parts(args, init=False)
  File "/home/pi/.pyenv/versions/3.7.2/lib/python3.7/pathlib.py", line 649, in _from_parts
    drv, root, parts = self._parse_args(args)
  File "/home/pi/.pyenv/versions/3.7.2/lib/python3.7/pathlib.py", line 633, in _parse_args
    a = os.fspath(a)
TypeError: expected str, bytes or os.PathLike object, not Asset

--- a/qrinvite/qrinvite.py
+++ b/qrinvite/qrinvite.py
@@ -50,7 +50,7 @@ class QRInvite(Cog):
             invite = invite.code
 
         if image_url is None:
-            image_url = ctx.guild.icon_url
+            image_url = str(ctx.guild.icon_url)
 
         if image_url == "":  # Still
             await ctx.send(

